README.md

```diff
@@ -3,15 +3,22 @@
   <img alt="Heroku Cloud Native Buildpacks Logo" src="./logo.png" width="100%">
 </p>
 
-## What are Heroku Cloud Native Buildpacks?
+Create a production-ready container image for your application with a [single command](TUTORIAL.md); no Dockerfile required. [Try Heroku Cloud Native Buildpacks today](TUTORIAL.md).
 
-Heroku Cloud Native Buildpacks, now in preview, can help you create a production ready container image for your app with a single command, without the need to write a Dockerfile or configuring a complex build system.
+## What is a Cloud Native Buildpack?
 
-Heroku Cloud Native Buildpacks implement the [Cloud Native Buildpacks specification](https://buildpacks.io/) (CNB) and are free and open-source software. They can be used independently of the Heroku platform.
+In 2012, Heroku introduced the world to the declarative build system now known as [classic buildpacks](https://blog.heroku.com/buildpacks). Twelve years later, we've joined forces with the Cloud Native Compute Foundation (CNCF) to bring the next evolution in building reliable and consistent images to everyone with [Cloud Native Buildpacks](https://buildpacks.io).
```

Comment:
Heroku joined forces with CNCF earlier than 2024 though? Maybe "Twelve years later, we've joined" can be rephrased with either "A few years ago, we joined..." or just "Since then, we've joined"?

Reply:
I like adding the link to the blog here.

The use of “classic” is an internal phrasing still, so needs some attention (just “known as buildpacks”)

Then I tend to agree with Rune that the date and time period don’t add much value.

Reply:
The goal is to establish a timeline and a pattern. I want to say/show:

We've been here for years, rocking our peers ([not putting suckers in fear](https://genius.com/Ll-cool-j-mama-said-knock-you-out-lyrics)). I.e. We're not following some random standard. We INVENTED the concept. This isn't a weekend hack project, these are the newest branches of an already strong tree. We open-sourced the original, and we're back at it. We are open-sourcing this one.

While we've been a part of CNCF for some time, I'm intentionally choosing to pick a milestone that gives us a larger number. The milestone of these preview releases. And I'm calling out the delta in time as I feel it's emotionally more weighted "twelve years" has an emotional impact, while asking people to do 2024-2012 math in their head does not.

This "call back to history" is a bit of a Heroku-ism and you can see it in a lot of our more grand blog posts. 

If you don't like the specific numbers, some other phrases that might convey similar emotional weight like "Over a decade ago" or "Heroku, founded nearly two decades ago, introduce the world to buildpacks..." (2007, so it's 3 years off, not sure if it's close enough for "nearly"). The goal is to convey: Permanance, weight, commitment to community, and leadership in the space.

> The use of “classic” is an internal phrasing still, so needs some attention (just “known as buildpacks”)

I think for current customers it's confusing to not have a name for both. I'm referring to them as "classic" here https://github.com/heroku/buildpacks-ruby/blob/main/docs/upgrading.md. Other terminology would be "v2" or "heroku specific" or "proprietary standard" which all seem not as well positioned as "classic." For the kubecon crowd, they won't need a differentiator, but I think for Heroku customers calling them "buildpacks" and "Cloud Native Buildpacks" is confusing.

Reply:
For "classic", we could say "the first" or "the original" or something similar. The usage is more like an adjective than a noun. I'm wanting some small acknowledgment that there's a difference between "<no descriptor> buildpacks" and "CNB buidpacks" (which now strikes me a being like saying "ATM machine").

Reply:
I'm not against saying that we've been working on (and indeed, invented) the concept of buildpacks for many years at all - or using this to show our commitment in this area.

It's not really about the numbers, I guess I just found it a bit odd to say that _"Twelve years later, we've joined forces with the Cloud Native Compute Foundation (CNCF)"_, when we actually [joined forces with them in 2018](https://blog.heroku.com/buildpacks-go-cloud-native) - if that makes sense?

Btw CNCF is "Cloud Native Computing Foundation" :)

Reply:
How about this:

```suggestion
Twelve years ago, Heroku introduced the world to the declarative build system known as [buildpacks](https://blog.heroku.com/buildpacks). Since then, we've joined forces with the Cloud Native Compute Foundation (CNCF) to bring the next evolution in building reliable and consistent images to everyone with [Cloud Native Buildpacks](https://buildpacks.io).
```